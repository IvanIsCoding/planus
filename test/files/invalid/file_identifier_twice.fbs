file_identifier "FOO";
file_identifier "BAR";