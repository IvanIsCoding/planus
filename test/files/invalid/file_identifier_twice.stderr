[0m[1m[38;5;9merror[0m[1m: Cannot set file_identifier twice[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/file_identifier_twice.fbs:2:1
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m file_identifier "FOO";
  [0m[34mâ”‚[0m [0m[34m----------------------[0m [0m[34mfirst declaration was here[0m
[0m[34m2[0m [0m[34mâ”‚[0m [0m[31mfile_identifier "BAR";[0m
  [0m[34mâ”‚[0m [0m[31m^^^^^^^^^^^^^^^^^^^^^^[0m [0m[31madditional declaration was here[0m

Error: Bailing because of errors
